File: 250218/숫자로 이루어진 사각형/rectangle-with-a-number.js
const fs = require("fs");

const n = Number(fs.readFileSync(0, "utf8").trim());

const printRect = n => {
    let cnt = 1;
    for (let i = 0; i < n; i++) {
        const line = [];
        for (let j = 0; j < n; j++) {
            line.push(cnt);
            cnt = cnt % 9 + 1;
        }
        console.log(line.join(" "));
    }
}

printRect(n);

File: 250218/숫자로 이루어진 사각형/rectangle-with-a-number.py
n = int(input())

def print_rect(n):
    cnt = 1
    for _ in range(n):
        line = []
        for _ in range(n):
            line.append(cnt)
            cnt = cnt % 9 + 1
        print(" ".join(map(str, line)))

print_rect(n)